addons/auth_signup/models/res_users.py

```diff
@@ -242,7 +248,7 @@ def create(self, vals_list):
             if users_with_email:
                 try:
                     users_with_email.with_context(create_user=True).action_reset_password()
-                except MailDeliveryException:
+                except UserError:
```

Comment:
@tde-banana-odoo hey, want to give this a quick glance? I don't think it should be an issue, it seems to make sense to just cancel the signup if there's any issue with reset_password instead of completely blocking user creation.

But UserError is pretty broad so... just checking

Reply:
(@tde-banana-odoo ding?)